src/Parser/FrostMage/Modules/Features/WintersChill.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatNumber } from 'common/format';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class WintersChillTracker extends Module {
+  static dependencies = {
+    enemies: Enemies,
+  };
+  totalWintersChill = 0;
+  inChillIceLance= 0;
+
+  on_byPlayer_cast(event) {
+    if (event.ability.guid !== SPELLS.ICE_LANCE_CAST.id) {
+      return;
+    }
+    const enemy = this.enemies.getEntity(event);
+    if (enemy.hasBuff(SPELLS.WINTERS_CHILL.id, event.timestamp)) {
+      this.inChillIceLance += 1;
+    }
+  }
+  on_applydebuff(event) {
```

Comment:
Should use `on_byPlayer_applydebuff(event)`

Reply:
Even though the debuff is on the target and not me, right?

Reply:
Yes, `onPlayer` is for debuffs applied on you (therefore you are the target of the spell), and `byPlayer` is for debuffs applied by you (you are the source of the spell).

Reply:
Applied to you is actually toPlayer, not onPlayer